src/app/onboarding/components/glam/glam-goals-screen/glam-goals-screen.component.scss

```
@@ -17,19 +21,27 @@
 }
 
 .chart-path {
-  font-family: 'UsualRegular', sans-serif;
+  font-family: 'UsualLight', sans-serif;
   font-size: 56px;
   width: 500px;
 
+  @media screen and (max-width: 1024px) {
+    width: 400px;
+  }
+
   @media screen and (max-width: 500px) {
     width: auto;
   }
 }
 
 .text {
-  font-family: 'UsualRegular', sans-serif;
+  font-family: 'UsualLight', sans-serif;
   font-size: 16px;
```

In general we should prefer `em/rem` units instead of `px`, especially for font size.
```suggestion
  font-size: 1rem;
```